### DISCUSSION TITLE
Slim Framework Driver

### DISCUSSION THREAD

**Discussion by rodmar35** [laravel/valet#1117] [Ideas]
Hi

I'm looking for a LocalValetDriver for SlimFramework : https://www.slimframework.com/
Can someone help me ?
thank you very much
RodMar

**Comment by MyNameIsAlaa** [comment]
hello, is there a solution for this? i'm also looking to run Slim framework with valet

**Comment by moussaclarke** [comment]
Slim 4 doesn't need anything fancy out of the box, you just map requests that aren't files/dirs onto `./public/index.php`, which the `BasicValetDriver` already handles - so this should just work. Try `valet which` from the site root to see which driver it would use.

If you need something custom, writing drivers is pretty straight forward, have a look at `~/.config/valet/Drivers/SampleValetDriver.php`
